redis/client.go

```diff
@@ -0,0 +1,32 @@
+package redis
+
+import (
+	"os"
+
+	"fmt"
+
+	"github.com/go-redis/redis"
+)
+
+//RedisClient is a *redis.Client that can be set to the same client you've created
+var RedisClient *redis.Client
+
+// NewStandardRedisClient generates a preconfigured redis client according to our spec.
+func NewStandardRedisClient(redisHostEnvironmentVariableName string) (client *redis.Client, err error) {
+	if redisHostEnvironmentVariableName == "" {
+		redisHostEnvironmentVariableName = "REDIS_HOST"
+	}
+
+	redisHost, present := os.LookupEnv("REDIS_HOST")
+	if !present {
+		err = fmt.Errorf("environment variable %s must be set in order to retrieve the redis hostname", redisHostEnvironmentVariableName)
```

Comment:
Have you seen [`init()`](https://golang.org/doc/effective_go.html#init) functions? That may be a better place for this check to live. If you are importing this package and using it, we can do an import-time check for `REDIS_HOST`.  You should be able to replace the `os.LookupEnv(string) (string, bool)` call with `os.Getenv(string) string`